projects/ngrx.io/content/guide/store/feature-creators.md

```diff
@@ -73,6 +73,72 @@ export const selectBookListPageViewModel = createSelector(
 );
 </code-example>
 
+`createFeature` also can be used to provide extra selectors for the feature state with the `extraSelectors` option:
+
+<code-example header="books.feature.ts">
+import { createFeature, createReducer, on } from '@ngrx/store';
+import { Book } from './book.model';
+
+import * as BookListPageActions from './book-list-page.actions';
+import * as BooksApiActions from './books-api.actions';
+
+interface State {
+  books: Book[];
+  query: string;
+}
+
+const initialState: State = {
+  books: [],
+  query: '',
+};
+
+export const booksFeature = createFeature({
+  name: 'books',
+  reducer: createReducer(
+    initialState,
+    on(BookListPageActions.search, (state, action) => ({
+      ...state,
+      query: action.query,
+    })),
```

Comment:
What about removing this (and the `on` import)?
It adds a bit of extra code while we want to show how `extraSelectors` work.

Reply:
I think I wanted to do as you said, but saw the previous examples have them. Did I miss something?

Reply:
We could also just add the extra selectors to the existing snippet 🤔
Otherwise it's just 90% the same

Reply:
I think I will just go with including it in the same snippet